dm/ctl/common/util.go

```diff
@@ -210,3 +211,22 @@ func IsDDL(sql string) (bool, error) {
 		return false, nil
 	}
 }
+
+// GetTaskNameFromArgOrFile tries to retrieve name field from the file if arg is filename-like, otherwise returns arg directly
+func GetTaskNameFromArgOrFile(arg string) string {
+	if !strings.HasSuffix(arg, ".yaml") {
```

Comment:
some people may name a YAML file with the suffix of `yml` rather than `yaml`. 😂

Reply:
done in https://github.com/pingcap/dm/pull/892/commits/6d1aede47719764e2eac05009255833c87f349c2

Reply:
we also need to update the master branch now.